apps/ima/views/cannot-use-form.html

```diff
@@ -0,0 +1,5 @@
+{{<partials-page}}
+  {{$page-content}}
+    <p>{{#t}}pages.cannot-use-form.paragraph-1{{/t}}</p>
+  {{/page-content}}
+{{/partials-page}}
```

Comment:
Wording needs to be changed according to latest Figma designs.

Reply:
done